web/iot/iot.js

```diff
@@ -110,72 +110,217 @@ function handleMessageHandler(payload){;
     else {
       statusBadge?.setAttribute('class', 'deviceStatusNone');
     }
+    handleStatusMessage(payload.message.message, payload.message.status);
   }
   catch(e){
     console.log(e);
   }
 }
 
+async function handleStatusMessage(message, status){
+  var statusSection = document.getElementById('status-message-section');
```

Comment:
I like the alerts popping up in the bottom right.  Can we somewhere convey that the user should check the chat app for more information?